### DISCUSSION TITLE
Sync photo vault with syncthing

### DISCUSSION THREAD

**Discussion by kapiteinroodbaard** [leonlatsch/Photok#238] [General]
Obviously by its nature all photos in Photok are encrypted. But would it be possible to use a file sync tool like syncthing to sync these encrypted photos to another device and decrypt them with my password? Having a photovault on multipe devices without depending on cloud would elevate Photok above all the other Photo Vault apps IMO.
If so where on the filesystem does Photok store its data?

**Comment by leonlatsch** [comment]
Great Idea, I really like this approach. However Photok certainly needs more dev time in order to make this possible.

Currently all files are stored inside the apps data directory, so not accessible for other apps. Also the encryption needs an overhaul #204 

Something simmillar has already been requested: #197 

But, I think this is the right direction for Photok. As you might have seen, I am currently doing very little development. Maybe add a comment to #197 or open a new issue :)

**Comment by kapiteinroodbaard** [comment]
For those finding this post in the future and looking for an alternative while we wait for this feature: you can use the app DroidFS combined with synching this way. It allows you to store, watch and sync photos and use Synching to sync the individual photos between devices.
I still feel Photok has a nicer 'gallery' style interface which DroidFS doesnt.
